docker-compose.yaml

```diff
@@ -733,7 +736,8 @@ services:
     ports:
       - "$ESPRESSO_BLOCK_EXPLORER_PORT:3000"
     environment:
-      - QUERY_SERVICE_URI:http://localhost:$ESPRESSO_SEQUENCER1_API_PORT/v0/
+      - QUERY_SERVICE_URI=http://localhost:$ESPRESSO_SEQUENCER1_API_PORT/v0/
```

Comment:
We are specifying the version here, which requires updating it every time we change the base version in the sequencer. Can we omit this and specify it on the client side? surf disco client takes a version and forwards it

Reply:
This is a configuration for the Block Explorer which doesn't use `rust` and cannot use `surf-disco` at all.  Anything not prefixed with the version results in a redirect status code.  So it seemss more simple to have the version suffixed.

Reply:
Gotcha